src/components/body/header/Header.css

```diff
@@ -1,8 +1,261 @@
-.nav-bar{
-    display: flex;
-    flex-direction: row;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
-    align-items: center;
+.side-bar {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 64px;
+  background: #FFFFFF;
+  border: 1px solid gray; /*for exam after delete border*/
```

Comment:
remove this comment - it is not easy to understand/ I do not see any reason to hold it...